.gitignore

```diff
@@ -1,8 +1,19 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
-docs/build/
-docs/site/
 Manifest.toml
 .DS_Store
 .vscode
+
+/Manifest.toml
```

Comment:
Manifest.toml (also below) and .vscode are duplicate.
We do not use `deps`, `tmp`, `generated` and `build` AFAICT. Why do you add them?

Reply:
- `tmp` so that my local tmp attempts don't get committed, `deps` should go
 - `generated` is also generated in local builds, and `build` should go also I think.